Resources/public/style/httplug.css

```diff
@@ -75,6 +75,13 @@
 }
 
 /**
+ * Stack
+ */
+.httplug-stack>.httplug-stack {
+    margin-left: 35px;
```

Comment:
Is `4rem` a value you want as 35px is not enough? I changed it to `2.5em` which is `35px` on a 96dpi screen.